### DISCUSSION TITLE
cant login into embedded terminal

### DISCUSSION THREAD

**Discussion by EnteSuesssauer15** [cockpit-project/cockpit#18107] [Q&A]
i dont really know what to say about this one, im using the terminal.html from the official website and i also get the login screen and when i do login i just get another login screen over and over again. i also tried making different users to try but with the same result

**Comment by marusak** [comment]
> im using the terminal.html from the official website

Can you please explain in details what steps you took?

Can you also take a look into journal if you see there any messages? Or in browser console?

**Reply by EnteSuesssauer15** [reply]
well i used the same exact code here https://cockpit-project.org/guide/latest/embedding and im trying to login with my credentials just like i do in putty but after i press on login it "refreshes" the page and im back to the login page. 

if the credentials are wrong, it will tell me its wrong.

![Bild_2023-01-02_131518515](https://user-images.githubusercontent.com/87829672/210229974-6efa96e2-5685-4b5b-a2b2-0616a0fb3f70.png)

thats what the journal says upon login 

Jan 03 00:10:11 homebridge cockpit-tls[16880]: cockpit-tls: gnutls_handshake failed: A TLS fatal alert has been received.
Jan 03 00:10:11 homebridge cockpit-ws[16885]: cockpit-ws: Failed to open certificate file /run/cockpit/tls/e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855: No such file or directory
Jan 03 00:10:11 homebridge cockpit-session[16945]: pam_ssh_add: Failed adding some keys
Jan 03 00:10:11 homebridge cockpit-session[16945]: pam_unix(cockpit:session): session opened for user frgd(uid=1000) by (uid=0)
Jan 03 00:10:11 homebridge systemd-logind[426]: New session 494 of user frgd.
Jan 03 00:10:11 homebridge systemd[1]: Started Session 494 of user frgd.
Jan 03 00:10:11 homebridge polkitd(authority=local)[405]: Registered Authentication Agent for unix-session:494 (system bus name :1.1316 [cockpit-bridge], object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_GB.UTF-8)
Jan 03 00:10:11 homebridge sudo[16960]:     frgd : PWD=/run/user/1000 ; USER=root ; COMMAND=/usr/bin/cockpit-bridge --privileged
Jan 03 00:10:11 homebridge sudo[16960]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
Jan 03 00:10:12 homebridge cockpit-tls[16880]: cockpit-tls: gnutls_handshake failed: A TLS fatal alert has been received.
Jan 03 00:10:12 homebridge cockpit-ws[16885]: cockpit-ws: Failed to open certificate file /run/cockpit/tls/e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855: No such file or directory
Jan 03 00:10:15 homebridge wpa_supplicant[449]: RRM: Ignoring radio measurement request: Not RRM network
Jan 03 00:10:23 homebridge wpa_supplicant[449]: RRM: Ignoring radio measurement request: Not RRM network
Jan 03 00:10:26 homebridge polkitd(authority=local)[405]: Unregistered Authentication Agent for unix-session:494 (system bus name :1.1316, object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_GB.UTF-8)
Jan 03 00:10:26 homebridge sudo[16960]: pam_unix(sudo:session): session closed for user root
Jan 03 00:10:26 homebridge cockpit-session[16945]: pam_unix(cockpit:session): session closed for user frgd
Jan 03 00:10:26 homebridge systemd[1]: session-494.scope: Succeeded.
Jan 03 00:10:26 homebridge systemd-logind[426]: Session 494 logged out. Waiting for processes to exit.
Jan 03 00:10:26 homebridge systemd-logind[426]: Removed session 494.

**Reply by EnteSuesssauer15** [reply]
so, i found this issue https://github.com/cockpit-project/cockpit/issues/14704
which seems like the same problem i have and when i saw at the bottom where it says its fixed in the new version ive tried to install cockpit via backports of debian which did work but im now unable to get the embedded terminal to work even tough it works when having the same link as the terminal in a new tab.

but the same reload problem persists

![Bild_2023-01-03_015715928](https://user-images.githubusercontent.com/87829672/210288191-142d9eed-35b7-4c27-ada5-8f3f3b41415a.png)

**Reply by EnteSuesssauer15** [reply]
i just installed a fresh copy on a different system with the same login problem but its only embedded, when using the same link in a new tab it works perfectly

**Reply by EnteSuesssauer15** [reply]
steps i took

1. open the website where the embedded terminal is 
2. enter my correct credentials
3. hit enter/login
4. back to the login screen with empty credential slots

**Reply by EnteSuesssauer15** [reply]
i tried it with chrome opera gx and edge none of them work but firefox does work
